miio/integrations/vacuum/roborock/vacuum.py

```diff
@@ -365,6 +369,41 @@ def map(self):
         # returns ['retry'] without internet
         return self.send("get_map_v1")
 
+    @command()
+    def get_multi_maps(self) -> MultiMapList:
+        """Return list of multi maps."""
+        if self._multi_maps is not None:
+            return self._multi_maps
+
+        self._multi_maps = MultiMapList(self.send("get_multi_maps_list")[0])
+        return self._multi_maps
+
+    @command()
+    def multi_map_enum(self) -> Optional[enum.Enum]:
+        """Enum of the available map names."""
+        if self._multi_map_enum is not None:
+            return self._multi_map_enum
+
+        multi_maps = self.get_multi_maps()
+
+        self._multi_map_enum = enum.Enum("multi_map_enum", multi_maps.map_name_dict)
+        return self._multi_map_enum
+
+    @command(click.argument("multi_map_id", type=int))
+    def load_multi_map(
+        self,
+        multi_map_enum: Optional[enum.Enum] = None,
+        multi_map_id: Optional[int] = None,
+    ):
+        """Change the current map used."""
+        if multi_map_enum is None and multi_map_id is None:
+            _LOGGER.error("Either multi_map_enum or multi_map_id is required.")
```

Comment:
Should this rather raise an exception if those are really required?

Reply:
Yes I changed it to a raise ValueError